.travis.yml

```diff
@@ -2,20 +2,23 @@ language: go
 
 matrix:
   include:
-    - go: 1.7.6
-    - go: 1.8.3
-    - go: tip
+    - go: 1.7.x
+    - go: 1.8.x
+    - go: 1.9.x
+    - go: master
   allow_failures:
-    - go: tip
+    - go: master
 
 before_install:
   - mkdir -p bin
   - curl -Lso bin/shellcheck https://github.com/caarlos0/shellcheck-docker/releases/download/v0.4.3/shellcheck
   - chmod +x bin/shellcheck
 script:
-  - PATH=$PATH:$PWD/bin go test -v ./...
```

Comment:
fyi, this line ignores the vendor folder as expected in go1.9, but not in previous versions of go.  The changes I've made work across all go versions.